client/src/main/java/com/battleships/start_window/InsertData/InsertSeekBarData.java

```diff
@@ -0,0 +1,40 @@
+package com.battleships.start_window.InsertData;
+
+import com.battleships.Translator;
+import javafx.fxml.FXML;
+import javafx.scene.control.Label;
+import javafx.scene.control.Slider;
+
+public class InsertSeekBarData {
+    @FXML
+    private Label rowsTextLabel;
+    @FXML
+    private Slider rowsProgressBar;
+    @FXML
+    private Label rowsAmountDisplayer;
+
+
+    @FXML
+    private Label columnsTextLabel;
+    @FXML
+    private Slider columnsProgressBar;
+    @FXML
+    private Label columnsAmountDisplayer;
+
+    public void initialize() {
```

Comment:
in FooterController initialize is annotated with @FXML, here it is not

Reply:
done